src/app/guards/authorize.guard.ts

```diff
@@ -54,13 +54,26 @@ export class AuthorizeGuard implements CanActivateChild {
     return this._user.getUserSession(queryParams).pipe(
       map((x) => x.oauthSession),
       map((session) => {
-        if (session.forceLogin || !oauthSessionUserIsLoggedIn(session)) {
-          return this.redirectToLoginPage(queryParams)
+        if (
+          session.redirectUrl &&
+          session.responseType &&
+          session.redirectUrl.includes(session.responseType)
```

Comment:
do we now all the possible values of `responseType`? Is it possible to add that to the types

Reply:
Ok, I will add them